game/CMakeLists.txt

```diff
@@ -195,7 +189,9 @@ endif()
 list(APPEND LIBS ${FFTW3_LIBRARIES})
 list(APPEND LIBS ${BLAS_LIBRARIES})
 
-target_link_libraries(performous PRIVATE fs ced aubio ${LIBS} PkgConfig::deps)
+find_package(Threads)
+
+target_link_libraries(performous PRIVATE fs ced aubio ${LIBS} PkgConfig::deps Threads::Threads)
```

Comment:
for now PkgConfig is alright, but on windows PkgConfig is hard. But that should be fixed in the native windows build pr

Reply:
was already there, BTW (just adding thread)

Reply:
Yeaa i know :p